amundsen_application/static/js/components/DashboardPage/index.tsx

```diff
@@ -171,7 +171,7 @@ export class DashboardPage extends React.Component<DashboardPageProps, Dashboard
                className="btn btn-default btn-lg">Open Dashboard</a>
           </div>
         </header>
-        <main className="column-layout-1">
+        <article className="column-layout-1">
```

Comment:
So does `axe` highlight issues like this, or did you just happen to know what these should be ideally be? I'd be curious to see how you use the tool to work through updating a page.

Reply:
Axe told me that there were a duplicated `<main>` tag after I added the global one on the top level component. 

Using an article here was my call on what would work better. You could [check here](https://www.smashingmagazine.com/2020/01/html5-article-section/) for more about the rationale.

With Axe, I just open its panel on the dev tools and see something like this:
![Fixes_DataSource_or_Dashboard_Icons_not_rendering_on_Firefox_by_Golodhros_·_Pull_Request__443_·_lyft_amundsenfrontendlibrary](https://user-images.githubusercontent.com/190833/81898821-4b455e80-956e-11ea-9008-62d2ebfd4eeb.png)

Then I circle through the issues (you can toggle the "Highlight" link on the top right to see which element its referring to) and check if I can fix them easily or would need more deep changes.